src/components/form/controls.tsx

```diff
@@ -150,6 +150,8 @@ export const SendButton = styled(Button)`
 
 export const Checkbox = styled.input`
   opacity: 0;
+  width: 1px;
+  height: 1px;
```

Comment:
I guess this is to visually hide the checkbox.
Common practices are using an empty clip path or setting opacity to 0. width/height are usually not used, although I don't know why. Why the addition to opacity? Did you have some layout shifts? then position absolute could save you too here. Again nothing to change if it works of course. Just asking for details

Reply:
Yes it worked with opacity to 0 and position: absolute 👍